typed/deduced_test.go

```diff
@@ -218,7 +218,7 @@ func TestMergeDeduced(t *testing.T) {
 func TestToSetDeduced(t *testing.T) {
 	tests := []objSetPair{
 		{`{"key":"foo","value":1}`, _NS(_P("key"), _P("value"))},
-		{`{"key":"foo","value":{"a": "b"}}`, _NS(_P("key"), _P("value", "a"))},
+		{`{"key":"foo","value":{"a": "b"}}`, _NS(_P("key"), _P("value"), _P("value", "a"))},
```

Comment:
Changes in this file clearly make sense ...

Reply:
Yes the changes here were all what I expected to see when changing the struct fields to map values by default